latest/docs-ref-conceptual/release-notes-azure-cli.md

```diff
@@ -16,6 +16,112 @@ ms.assetid: ce0428f7-0a59-4e72-9237-d907b171af51
 
 # Azure CLI 2.0 release notes
 
+## November 14, 2017
+
+Version 2.0.21
+
+### ACR
+
+* Added support for creating webhooks in replication regions
+
+
+### ACS
+
+* Changed all wording of "agent" to "node" in AKS
+* Deprecated `--orchestrator-release` option for `acs create`
+* Changed default VM size for AKS to `Standard_D1_v2`
+* Fixed `az aks browse` on Windows
+* Fixed `az aks get-credentials` on Windows
+
+### Appservice
+
+* Added deployment source `config-zip` for webapps and function apps
+* Added `--docker-container-logging` option [1]
+* Removed the `--storage` option from `--web-server-logging` [2]
+* Improved error messages for `deployment user set`
+* Added support for creating Linux function apps
+* Fixed `list-locations`
+
+### Batch
+
+* Fixed bug in pool create command when a resource ID was used with the `--image` flag
+
+### Batchai [3]
```

Comment:
The listed "options" in this section need to be named.

Reply:
addressed.